docs/content/en/tasks/migrate/_index.md

```diff
@@ -6,227 +6,245 @@ weight: 1
 
-### Update replication strategies
+Change the values to match your deployment settings, knowing that the Cassandra service is named as follows by K8ssandra: `<cluster name>-<dc name>-service`.
 
-Next we need to stop replicating data to nodes in the old datacenter. For each keyspace that was previously updated, we need to update it again. This time we will remove the old datacenter. 
+## Restrict traffic to dc2
+Client traffic should be restricted to dc2 by selecting it as local datacenter and using only `LOCAL_` consistency levels (or `QUORUM`). This will prevent nodes from connecting to `dc1` as we decommission it.
 
-Here is an example that specifies the `system_auth keyspace`:
+## Decommission dc1
+Update the K8ssandraCluster object definition by removing the externalDatacenters and additionalSeeds sections. Only after that, alter all keyspaces to remove replicas from dc1 (if you do it before, the operator will re-alter some keyspaces replication):
 
-```cql
-ALTER KEYSPACE system_auth WITH replication = {'class': 'NetworkTopologyStrategy', 'dc2': 3}
+```
+ALTER KEYSPACE <my-keyspace> WITH replication = {'class': 'NetworkTopologyStrategy', 'dc2': 3};
 ```
 
-## Remove the old datacenter
-
-Now we are ready to do some cleanup and remove the old datacenter and associated resources that are no longer in use. There are three steps:
-
-1. Decommission nodes
-2. Remove old seed nodes
-3. Delete old datacenter resources from Kubernetes
+Stop all nodes from `dc1` by deleting its helm release (in the case of a K8ssandra 1.x cluster) or stopping all Cassandra processes (in the case of a non-k8s Cassandra cluster).
 
-### Decommission nodes
+Connect to one of the Cassandra pods in the Cassandra container, and remove nodes from `dc1` one by one using their host id which shows up in the `nodetool status` output.
 
-Run the following on each node in the old datacenter:
+In our example, we will run the following commands:
 
-```bash
-nodetool decommission
 ```
+$ nodetool -u <superuser-name> -pw <superuser-password> removenode 9299e226-d2fa-4c2e-9fba-3b344d5d47b7
 
-{{% alert title="Note" color="success" %}}
-Decommissioning should be done serially, one node at a time.
-{{% /alert %}}
+$ nodetool -u <superuser-name> -pw <superuser-password> removenode 51c2fb0e-7fad-4b6e-8e4d-2108dc204cd5
 
-### Remove the old seed nodes
+$ nodetool -u <superuser-name> -pw <superuser-password> removenode 6a92531f-a086-4c40-9067-50899b8974a0
+```
 
-Remove the seeds nodes from the `additionalSeeds` chart property. You can simply remove the `additionaSeeds` property from the chart overrides file (see again the sample file shown in the [Install K8ssandra](#install-k8ssandra) section above). Edit your version of the values file, and then run a `helm upgrade` for the changes to take effect. Assuming that the edited chart properties are stored in `k8ssandra-values.yaml`:
+The `nodetool status` output should then be:
```

Comment:
I think at this point, you need to re-configure Reaper again and remove the dc1 seed service, wdyt?